projects/expat/build.sh

```diff
@@ -15,6 +15,8 @@
 #
 ################################################################################
 
+export LDFLAGS="${CFLAGS}"  # for AddressSanitizer
```

Comment:
nit: I'd fix the comment to say "to make sure we link with sanitizer runtime"

Reply:
Alright, no problem. Done.